src/main/kotlin/com/buildit/bookit/v1/bookable/BookableController.kt

```diff
@@ -12,18 +12,15 @@ import org.springframework.web.bind.annotation.RestController
  */
 @RestController
 @RequestMapping("/v1/bookable")
```

Comment:
maybe not related to this PR, but were we going to change this to /v1/location/<location>/bookable?

Reply:
I want to hold off on this one until we get locations sorted out